<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>544</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="2">
       <widget class="QScrollBar" name="horizontalScrollBar">
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBoxPar">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
        <property name="title">
         <string>Parameters</string>
        </property>
        <widget class="QLabel" name="labelNumOfVillages">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>57</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>Villages</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelNumOfHh">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Households</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="textEditN">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>30</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
        <widget class="QTextEdit" name="textEditH">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>50</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
        <widget class="QTextEdit" name="textEditR">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>70</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
        <widget class="QLabel" name="labelNumOfR">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Routes</string>
         </property>
        </widget>
        <widget class="QPushButton" name="buttonPlot">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>110</y>
           <width>81</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#buttonPlot{
background: rgb(238, 238, 238);
}</string>
         </property>
         <property name="text">
          <string>plot graph</string>
         </property>
        </widget>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>209</x>
           <y>0</y>
           <width>191</width>
           <height>141</height>
          </rect>
         </property>
         <property name="title">
          <string>Solution</string>
         </property>
         <widget class="QPushButton" name="pushButtonInitialSolution">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>101</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Initial Solution</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item row="0" column="2" rowspan="3">
       <widget class="QCustomPlot" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">#widget {
border: 2px solid gray;
border-radius: 10px;
background: white;
}</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="3">
       <widget class="QScrollBar" name="verticalScrollBar">
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBoxPlotSetting">
        <property name="styleSheet">
         <string notr="true">#groupBoxPlotSetting {
border: 2px solid gray;
border-radius: 10px;
}</string>
        </property>
        <property name="title">
         <string>Plot Settings</string>
        </property>
        <widget class="QCheckBox" name="checkBoxVillageNames">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>191</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Show names of the villages</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBoxShowRoads">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Show Roads</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBoxRainingSeazon">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>131</width>
           <height>21</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Raining Season</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#frame {
border: 2px solid gray;
border-radius: 10px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="buttonOpenVillages">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>20</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Villages...</string>
         </property>
        </widget>
        <widget class="QLabel" name="labFile1">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>....</string>
         </property>
        </widget>
        <widget class="QLabel" name="labFile2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>....</string>
         </property>
        </widget>
        <widget class="QLabel" name="labFile3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>....</string>
         </property>
        </widget>
        <widget class="QPushButton" name="buttonOpenRoads">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>70</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Roads...</string>
         </property>
        </widget>
        <widget class="QPushButton" name="buttonOpenHouseh">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>110</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Households...</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>883</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionVillages">
   <property name="text">
    <string>Villages</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
